### PR TITLE
update envoy

### PR DESCRIPTION
**Pull request by yuval-k**
# Description
updated envoy

### CODE REVIEW

File: changelog/v1.3.31/bump-envoy.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+changelog:
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: envoy-gloo
+    dependencyRepo: solo-io
+    dependencyTag: v1.14.3-patch1
+    description: >
+      Updated envoy-gloo to one based on enovy 1.14.3, which includes a security release.
```
Comment (kdorosh) [inline changelog/v1.3.31/bump-envoy.yaml:7]:
```suggestion
      Updated envoy-gloo to one based on envoy 1.14.3, which includes a security release. 
```

Comment (kdorosh) [reply changelog/v1.3.31/bump-envoy.yaml:7]: also which security release? as a user i'd want to know

Comment (yuval-k) [reply changelog/v1.3.31/bump-envoy.yaml]: not sure what you mean? 1.14.3 is the security release

**Pull request merged**